src/components/canvas-tools/templates/Line.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div>
+    <v-color-picker
```

Comment:
@Eirmas ,Can you update this PR as well with your imported colour picker and slider?